tasks/65_smaller-common-multiple/smaller-common-multiple.js

```diff
@@ -0,0 +1,16 @@
+export function smallestCommons(arr) {
```

Comment:
take a look at Euclidean Algorithm it will easier and probably eat fewer resources